learn/framework/index.md

```diff
@@ -1,9 +1,22 @@
 ---
 title: Appchain Development Framework
 description: Learn about the appchain Development Framework, Rust Programming Language, Substrate and its features, including XCM and ready-to-use pallets for your Runtime.
-template: subsection-index-page.html
-hide: 
- - toc
- - feedback
+template: index-page.html
 ---
 
+# Appchain Development Framework
+
+Building a blockchain from scratch is an intricate process, demanding deep knowledge of consensus algorithms, cryptography, data structures, governance, and more. With Tanssi, you can bypass these complexities and focus entirely on your vision. The Appchain Development Framework offers a streamlined approach to creating tailored blockchains, leveraging Substrate's powerful architecture to handle every essential internal component of a blockchain.
+
+Designed for flexibility, this framework allows you to adapt runtime characteristics, integrate pre-built modules, or create new ones based on your appchain’s specific needs. Whether it’s enhancing core functionality, optimizing for scalability, or enabling seamless user interactions, Tanssi provides the foundation to bring your blockchain ideas to life.
+
+Key Features of the Framework:
+
+- **Substrate Core Architecture:** Divided into the **Core Client**, which manages external communications and internal responsibilities like storage, and the **Runtime**, which executes transactions and manages state transitions with customizable logic
```

Comment:
Here, we don't need to put Core Client and Runtime in bold. In most cases, we'll use italics to draw attention to specific words and we leave bold for things that are super important, must-follow type things where we're sort of like HEY! PAY ATTENTION! haha

Also, no need to capitalize either of these words. The only time we really capitalize a word mid-sentence is like in link text (but only when the link text is the name of the page that we're linking to), or if it's a product name and always gets capitalized